src/game/Coordinate.java

```diff
@@ -1,6 +1,4 @@
-package piece;
-
-import game.BoardState;
+package game;
```

Comment:
Moved Coordinate in the game package, since it is used in conjunction with board and pieces and not exclusively by piece.